README.md

```diff
@@ -369,3 +370,69 @@ Then you can use [`pdb` commands](https://docs.python.org/3/library/pdb.html#deb
 `/opt/venvs/securedrop-client/lib/python3.7/site-packages/securedrop_client/`.
+
+Sometimes there is a bug in Qt rather than the client, so it helps to install [a debug version of PyQt5](#build-and-install-a-debug-version-of-pyqt5) which will allow you to see Qt debug symbols in your Python tracebacks. You can use `gdb` to view a traceback as well as a core file, if you're debugging a segfault, but first you must need to install the following some additional packages in order for `gdb` to work with Python programs.
+
+```
+sudo apt-get install libclang-dev '^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev gperf bison flex libgl1-mesa-dev pkg-config libdbus-1-dev libnss3-dev libfontconfig1-dev libxi-dev libxcomposite-dev libxcursor-dev libxtst-dev python3-sip-dev
+```
+
+### Build and install a debug version of PyQt5
+
+1. Build a debug version of Qt 5
+
+Clone the Qt repo and build it from source following these instructions: https://wiki.qt.io/Building_Qt_5_from_Git#Getting_the_source_code. This defaults to creating a 'debug' build and installs the binaries in the current directory, avoiding the need for `make install`. Note that the `-developer-build` option causes more symbols to be exported in order to allow more classes and functions to be unit tested than in a regular Qt build.
+
+```
+git clone https://code.qt.io/qt/qt5.git
+cd qt5
+git checkout v5.11.3                                # or checkout a different version
+git submodule update --init --recursive
+export LLVM_INSTALL_DIR=/usr/llvm
+mkdir ../qt5-build && cd ../qt5-build               # we don't want to build in the source code directory
+../qt5/configure -developer-build -opensource -nomake examples -nomake tests -confirm-license
+make -j 4
+```
+
+2. Prepare to build PyQt5 
+
+We are going to use the client virtual environment to build and install PyQt5. Make sure the client dependencies are installed. This will include `pyqt5-sip` which will be used when we build pyqt5 from source.
+
+If you haven't already, download the client and set up the virtual environment. 
+
+```
+git clone git@github.com:freedomofpress/securedrop-client.git
+cd securedrop-client
+virtualenv --python=python3.7 .venv
+source .venv/bin/activate
+pip install --require-hashes -r dev-requirements.txt
+```
+
+Make sure to uninstall the prepackaged pyqt5 library since we will be using our own.
+
+```
+pip uninstall pyqt5
+```
+
+3. Download the PyQt5 source tarball from sourceforge](https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.11.3/PyQt5_gpl-5.11.3.tar.gz/download) (the source tarball is not on PyPI, and riverbankcomputing does not make the source code for previous non-dev releases available)
```

Comment:
@kushaldas any idea where we can access to the SCM for this project? PyPI only hosts the binary wheels for most versions https://pypi.org/project/PyQt5/5.11.3/#files and https://www.riverbankcomputing.com/software/pyqt/ only hosts the latest dev snapshot